### PR TITLE
More themes: westeros, wonderland, chalk, halloween, essos,  walden and purple-passion

### PR DESCRIPTION
**Pull request by chfw**
本次 PR 内容，

## walden

screen shot 2018-08-11 at 23 51 40
screen shot 2018-08-11 at 23 51 20
screen shot 2018-08-11 at 23 51 05

## westeros

screen shot 2018-08-11 at 23 56 07
screen shot 2018-08-11 at 23 51 40
screen shot 2018-08-11 at 23 57 16

### CODE REVIEW

**Comment by chfw** [comment]
## essos

screen shot 2018-08-11 at 23 58 44
screen shot 2018-08-11 at 23 59 27
screen shot 2018-08-11 at 23 59 49

**Comment by chfw** [comment]
## halloween

screen shot 2018-08-12 at 00 00 57
screen shot 2018-08-12 at 00 01 27
screen shot 2018-08-12 at 00 01 42

**Comment by chfw** [comment]
## chalk

screen shot 2018-08-12 at 00 02 48
screen shot 2018-08-12 at 00 02 41
screen shot 2018-08-12 at 00 02 27

**Comment by chfw** [comment]
## purple-passion

screen shot 2018-08-12 at 00 05 36
screen shot 2018-08-12 at 00 05 19
screen shot 2018-08-12 at 00 05 03

**Comment by chfw** [comment]
## wonderland

screen shot 2018-08-12 at 00 07 04
screen shot 2018-08-12 at 00 06 56
screen shot 2018-08-12 at 00 06 42

File: pyecharts/base.py [hunk 1]
```diff
@@ -76,14 +76,9 @@ def page_title(self):
...
     def use_theme(self, theme_name):
-        if theme_name in constants.ALL_THEMES:
-            self.theme = theme_name
-            if theme_name in constants.EXTERNAL_THEMES:
-                self._js_dependencies.add(self.theme)
-        else:
-            raise exceptions.InvalidTheme(
-                "{0} is not found".format(theme_name)
-            )
+        self.theme = theme_name
```
Comment (chenjiandongx) [inline pyecharts/base.py]: 👍

**Comment by chenjiandongx** [comment]
@chfw 
image

**Comment by chfw** [comment]
## romantic

screen shot 2018-08-13 at 10 49 53
screen shot 2018-08-13 at 10 41 56
screen shot 2018-08-13 at 10 51 46

**Pull request merged**